AutoRest/AutoRest.Core/ClientModel/Method.cs

```diff
@@ -59,6 +60,22 @@ public Method()
         public List<Parameter> Parameters { get; private set; }
 
         /// <summary>
+        /// Gets the list of Parameter Expansions
+        /// </summary>
+        public Dictionary<string, Dictionary<Property, Parameter>> ParameterExpansions { get; private set; }
```

Comment:
We should also add a `public Parameter Body {get;}` to get easier access to the body parameter